lib/mintel/dashboards/services/omni/omni-web-overview.libsonnet

```diff
@@ -10,6 +10,7 @@ local django = import 'components/panels/django.libsonnet';
 local celery = import 'components/panels/celery.libsonnet';
 local webService = import 'components/panels/frontend-service.libsonnet';
 local containerResources = import 'components/panels/container_resources.libsonnet';
+local omniWeb = import 'analytics.libsonnet';
```

Comment:
should we call this `omni-web-analytics.libsonnet` (just because its related to metrics from omni-web, rather than the analytics-service.

Reply:
yeah

Reply:
It's used for both so yes makes sense.  Updated.